### ISSUE TITLE
Autoptsserver timeout

### ISSUE THREAD

**Issue by koffes** [auto-pts/auto-pts#1308]
If `autoptsserver` is running, a test is started, and there is no PTS dongle connected, PTS will timeout and then restart indefinitely.

**Comment by sjanc** [comment]
Agree, it would be much easier to spot setup error if PTS server would simply refuse connection in such case and log error
